src/Tooling/Bit.Tooling.SourceGenerators/AutoInject/AutoInjectAttribute.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace Bit.Tooling.SourceGenerators;
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
I'd prefer ms.ext.di.abstractions, because our approach works well with all implementations of ms.ext.di.abstractions such as autofac.ext.di